web/styles/settings.css

```diff
@@ -1272,7 +1272,7 @@ label.preferences-radio-choice-label {
 #settings_page {
     height: 95vh;
     width: 97vw;
-    max-width: 1024px;
+    max-width: 73.1428em;
```

Comment:
This conversion is probably okay, but I'd like to have the pixel value for the em notated here.

Reply:
Oops, thanks for noticing